### ISSUE TITLE
Missing h1 for organisation page in platform admin

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#1240]
## Describe the bug 

There is no h1 on this screen: https://staging.notification.cdssandbox.xyz/organisations

## [Bug Severity](https://docs.google.com/document/d/1tcD32bjJRaREsMgaUEoXAk400y1dKzzaKR0G5Sr1w20/edit)
See examples in the documentation

SEV-4

## To Reproduce

1. Go to platform admin
2. Under Organization
3. Notice no heading level 1

## Expected behavior

We should have heading 1 to identify all sections in the platform admin. 

## Impact

Especially now that we have platform admins outside of CDS, we might not be able to support or help admins that can't reach out to us as easily. The admin tools should get the same attention as the rest of Notify. 

If applicable

Impact on Notify users: PTM platform admins

Impact on Recipients: None

Impact on Notify team: We need to know what we are doing, and potentially support others on our team. Admin is not the friendliest part of the UI. 

## Screenshots

![image.png](https://images.zenhubusercontent.com/614b439e980ccd4137400d3e/249a1f36-46f3-4791-a671-e6e259bfc0b8)

**Comment by YedidaZalik** [comment]
Omit from Ally stmt as only Notify staff affected
